docs/jj_first_principles.md

```diff
@@ -0,0 +1,50 @@
+# Jujutsu from first principles
+
+## Preface
+
+Why does Jujutsu exist and which problems does it solve? This document tries to
+answer both of these questions while expanding on the design in a user-friendly
+way. 
+
+At its core Jujutsu is [Version Control System][vcs] which scales to huge 
+repositories at [Google scale][billion-lines]. Many design choices are 
+influenced by the concurrent commits happening in Googles Monorepo, as there 
+are always multiple people working on the same file(s) at the same time.
+
+## Core Tenets
+
+Jujutsu's core tenets are:
+
+ 1. Having as few states as possible.
+ 1. Make it fundamentally impossible to lose work in your repository.
+ 1. Allow concurrent edits on any commit, pending or finished.
+ 1. Make a "stacked diffs" workflow as easy as possible.
+
+## Base design
+
+The initial base design is to be a conceptually simpler Mercurial, as 
```

Comment:
I would consider moving this sentence to near the top, as one of the very first ones. And possibly change it to something like

> The initial base design is to be a conceptually simpler Mercurial, without losing any of the powerful features DVCSes like Mercurial and Git have.

A lot of git folks worry a lot that "simpler" means "I can't do what I need to do," and also think already that Mercurial is a simpler git, so this would help address that.

Just my two cents though :)

Reply:
Thanks for the great suggestion. 

> A lot of git folks worry a lot that "simpler" means "I can't do what I need to do," and also think already that Mercurial is a simpler git, so this would help address that.

That's another great point, although I'm trying to avoid any references to Git as it is written from a pure Jujutsu PoV.

Reply:
I agree that being from a pure jujutsu POV makes sense, but you're already referencing Mercurial in this sentence, so I figured it was okay. :D